src/main/java/uk/gov/ons/census/casesvc/service/AddressTypeChangeService.java

```diff
@@ -98,6 +98,10 @@ private void createNewCase(
     newCase.setHtcWillingness(oldCase.getHtcWillingness());
     newCase.setHtcDigital(oldCase.getHtcDigital());
 
+    if (newCase.getCaseType().equals("CE")) {
```

Comment:
Why only do this for CE cases? Are we taking the wrong approach completely. We could change `ceActualResponses` from an `Integer` to an `int` and save ourselves headache.

Could do the same with `ceExpectedCapacity` - make it an `int` and save ourselves a bunch of weird null pointer exceptions.